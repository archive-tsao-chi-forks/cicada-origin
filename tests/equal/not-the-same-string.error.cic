the(Equal(String, "ratatouille", "baguette"), same)
