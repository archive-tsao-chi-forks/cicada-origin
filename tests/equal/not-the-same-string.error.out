
I am expecting the following two values to be the same type:
  String
But they are not.
from:
  "ratatouille"
to:
  "baguette"

previous:
- same
- @the Equal(String, "ratatouille", "baguette") same


